packages/react-core/src/demos/DataListDemo.md

```diff
@@ -418,3 +424,899 @@ class ExpandableDataList extends React.Component {
+    );
+  }
+
+  getAllItems() {
+    const { res } = this.state;
+    const collection = [];
+    for (const items of res) {
+      collection.push(items.id);
+    }
+
+    return collection;
+  }
+
+  updateSelected() {
+    const { res, selectedItems } = this.state;
+    let rows = res.map((post) => {
+      post.selected = selectedItems.includes(post.id);
+      return post;
+    });
+
+    this.setState({
+      res: rows
+    });
+  }
+
+  fetch(page, perPage) {
+    fetch(`https://my-json-server.typicode.com/jenny-s51/cardviewdata/posts?_page=${page}&_limit=${perPage}`)
+      .then((resp) => resp.json())
+      .then((resp) => this.setState({ res: resp, perPage, page }))
+      .then(() => this.updateSelected())
+      .catch((err) => this.setState({ error: err }));
+  }
+
+  componentDidMount() {
+    this.fetch(this.state.page, this.state.perPage);
+  }
+
+  renderPagination() {
+    const { page, perPage, totalItemCount } = this.state;
+
+    const defaultPerPageOptions = [
+      {
+        title: '1',
+        value: 1
+      },
+      {
+        title: '5',
+        value: 5
+      },
+      {
+        title: '10',
+        value: 10
+      }
+    ];
+
+    return (
+      <Pagination
+        itemCount={totalItemCount}
+        page={page}
+        perPage={perPage}
+        perPageOptions={defaultPerPageOptions}
+        onSetPage={(_evt, value) => {
+          this.fetch(value, perPage);
+        }}
+        onPerPageSelect={(_evt, value) => {
+          this.fetch(1, value);
+        }}
+        variant="top"
+        isCompact
+      />
+    );
+  }
+
+  buildSelectDropdown() {
+    const { splitButtonDropdownIsOpen, selectedItems, areAllSelected } = this.state;
+    const numSelected = selectedItems.length;
+    const allSelected = areAllSelected;
+    const anySelected = numSelected > 0;
+    const someChecked = anySelected ? null : false;
+    const isChecked = allSelected ? true : someChecked;
+    const splitButtonDropdownItems = (
+      <>
+        <DropdownItem key="item-1" onClick={this.selectNone.bind(this)}>
+          Select none (0 items)
+        </DropdownItem>
+        <DropdownItem key="item-2" onClick={this.selectPage.bind(this)}>
+          Select page ({this.state.perPage} items)
+        </DropdownItem>
+        <DropdownItem key="item-3" onClick={this.selectAll.bind(this)}>
+          Select all ({this.state.totalItemCount} items)
+        </DropdownItem>
+      </>
+    );
+    return (
+      <Dropdown
+        onSelect={this.onSplitButtonSelect}
+        isOpen={splitButtonDropdownIsOpen}
+        onOpenChange={(isOpen) => this.setState({ splitButtonDropdownIsOpen: isOpen })}
+        toggle={(toggleRef) => (
+          <MenuToggle
+            ref={toggleRef}
+            isExpanded={splitButtonDropdownIsOpen}
+            onClick={this.onSplitButtonToggle}
+            aria-label="Select cards"
+            splitButtonOptions={{
+              items: [
+                <MenuToggleCheckbox
```

Comment:
MenuToggleCheckbox needs to be added as an import around line 400. Along with SelectOption and Divider.